Tests/test_ProtParam.py

```diff
@@ -14,59 +15,62 @@
 
 
 class ProtParamTest(unittest.TestCase):
+    """Tests for ProtParam."""
+
     def setUp(self):
+        """Initialise objects."""
         self.seq_text = "MAEGEITTFTALTEKFNLPPGNYKKPKLLYCSNGGHFLRILPDGTVDGTRDRSDQHIQLQLSAESVGEVYIKSTETGQYLAMDTSGLLYGSQTPSEECLFLERLEENHYNTYTSKKHAEKNWFVGLKKNGSCKRGPRTHYGQKAILFLPLPV"
         self.analysis = ProtParam.ProteinAnalysis(self.seq_text)
 
     def test_count_amino_acids(self):
-        "Test getting amino acid counts"
+        """Calculating amino acid counts."""
```

Comment:
So we don't enforce "imperative mood" in ```Tests```?

Reply:
Not yet, but we should. I was initially running all the desired flake8 checks on the files I changed, but ran out of energy. Well spotted, that was an unfortunate rewording on my part...